content/swagger/parameters.yaml

```diff
@@ -189,3 +189,10 @@ with_enriched_attributes:
   required: false
   type: boolean
   x-from-version: "SaaS"
+with_root_parent:
+  name: with_root_parent
+  in: query
+  description: Retrieve the root parent product model code of a variant
```

Comment:
For consistency with the rest of the doc, I think `Return` is better than `Retrieve`.